default.json

```diff
@@ -235,6 +236,14 @@
       "minimumReleaseAge": "0 days",
       "updateNotScheduled": true
     },
+    {
+      "matchManagers": ["npm"],
+      "matchDepNames": ["^@seek/*"],
+      "matchUpdateTypes": ["major", "minor", "patch"],
+      "schedule": "before 3:00 am every weekday",
+      "minimumReleaseAge": "0 days",
+      "updateNotScheduled": true
+    },
```

Comment:
Is this meant to be the same as https://github.com/seek-oss/rynovate/blob/8f6d1b49e338cb24a343111edf0dc99294400033/default.json#L150-L158

It's possible there's an issue with specifying both `matchDepNames` and `matchPackagePatterns`, and we should probably migrate all of the latter to the former across the configuration.

Reply:
Ooo I think we want `packageName` so `matchPackageName`

https://github.com/renovatebot/renovate/discussions/14885#discussion-3978348